docs/examples/test_examples.py

```diff
@@ -0,0 +1,268 @@
+""" Tests for the files in nameko/docs/examples
```

Comment:
It's tricky to have this file as ``test/test_examples.py`` and also check coverage of the ``docs/examples`` directory. I've settled on giving it a separate entry in the Makefile and adding it as one of the steps under ``make test``.